src/playercards/customizable/UpgradeSheetLibrary.ttslua

```diff
@@ -120,27 +140,21 @@ function createRowCheckboxes(rowIndex)
     local func = function() clickCheckbox(rowIndex, col) end
     self.setVar(funcName, func)
     local checkboxPos = getCheckboxPosition(rowIndex, col)
-
-    self.createButton({
-      click_function = funcName,
-      function_owner = self,
-      position       = checkboxPos,
-      height         = boxSize * 10,
-      width          = boxSize * 10,
-      font_size      = 1000,
-      scale          = { 0.1, 0.1, 0.1 },
-      color          = { 0, 0, 0 },
-      font_color     = { 0, 0, 0 }
-    })
+    local identifier = tostring(rowIndex) .. " " .. tostring(col)
+    local newCheckbox = {
+      tag = "Button",
+      attributes = {
+        onClick = funcName,
+        position = checkboxPos,
+        id = identifier,
+      }
+    }
+    table.insert(xmlTable, newCheckbox)
   end
 end
 
 function getCheckboxPosition(row, col)
-  return {
-    x = xInitial + col * xOffset,
-    y = Y_VISIBLE,
-    z = customizations[row].checkboxes.posZ
-  }
+  return tostring((xInitial + col * xOffset) * -100) .. " " .. tostring(customizations[row].checkboxes.posZ * 100) .. " " .. tostring(-22)
```

Comment:
Nitpick: Split this into multiple separate statements (and give them useful names) to make this easier on the next person that works on it.
Also `tostring(-22)` should just be `"-22"`.